Makefile

```diff
@@ -33,23 +33,23 @@ clean-test:
 	rm -fr htmlcov/
 
 lint:
-	flake8 bin/* hg_agent_periodic tests
+	flake8 bin hg_agent_periodic tests setup.py
```

Comment:
We will have a folder with virtualenv on CI. That was the reason why I've added a strict list of packages. flake8 is starting checking files in virtualenv folder and it takes "ages" to complete. But you're right, it might be better to exclude virtualenv folder rather than adding the full list of files.